src/gsi/berror.f90

```diff
@@ -385,7 +383,6 @@ subroutine set_predictors_var
 
```

Comment:
While compiling in debug mode, ostats (line 378) and ostats_t (line 379) were identified as unused.  Please remove unused variables.